test/GoodGhostingGasEstimate.test.js

```diff
@@ -184,8 +211,8 @@ contract("GoodGhosting", (accounts) => {
             }
         });
 
-        it("admin cannot withdraw with 0% fees", async () => {
-            await truffleAssert.reverts(goodGhosting.adminFeeWithdraw({ from: admin }), "No Fees Earned");
-        })
+        it("admin withdraws admin fee from contract", async () => {
+                await truffleAssert.reverts(goodGhosting.adminFeeWithdraw({ from: admin }), "No Fees Earned");
+        });
```

Comment:
The previous suggestion would also handle both scenarios - admin fees equal to zero, or greater than 0.
By reverting it back, now we can only get gas estimating for when the admin fee is zero.

Reply:
yeah will revert